LSPDFR+/LSPDFRPlusHandler.cs

```diff
@@ -48,7 +48,43 @@ public static void Initialise()
             try
             {
                 EnhancedTrafficStop.BringUpTrafficStopMenuControllerButton = ini.ReadEnum<ControllerButtons>("General", "BringUpTrafficStopMenuControllerButton", ControllerButtons.DPadRight);
-                EnhancedTrafficStop.BringUpTrafficStopMenuKey = (Keys)kc.ConvertFromString(ini.ReadString("General", "BringUpTrafficStopMenuKey", "E"));
+                EnhancedTrafficStop.BringUpTrafficStopMenuKey = (Keys)kc.ConvertFromString(ini.ReadString("General", "BringUpTrafficStopMenuKey", "D7"));
+
+
+                Keys stockTrafficStopInteractKey = Keys.E;
+                Keys stockTrafficStopInteractModifierKey = Keys.None;
+                ControllerButtons stockTrafficStopInteractControllerButton = ControllerButtons.DPadRight;
+                try
+                {
+                    InitializationFile stockLSPDFRIni = new InitializationFile(Path.Combine("lspdfr", "keys.ini"));
+                    stockTrafficStopInteractKey = stockLSPDFRIni.ReadEnum<Keys>("", "TRAFFICSTOP_INTERACT_Key", Keys.E);
```

Comment:
These lines don't actually work and always return the default given value! Seems that RPH inis have trouble reading from INI files without a section.